### ISSUE TITLE
Remove obviously rubbish tokens in preprocessing

### ISSUE THREAD

**Issue by Twigonometry** [Twigonometry/Phishing-ML#3]
Current output of most common words:

```
[('_', 52195),
 ('â', 9823),
 ('½ï', 7960),
 ('subject', 6630),
 ('com', 6140),
 ('1', 5574),
 ('email', 5353),
 ('e', 5095),
 ('http', 4930),
 ('2', 4520),
 ('free', 4512),
 ('3', 4315),
 ('000', 4127),
 ('mail', 3952),
 ('please', 3950),
 ('money', 3929),
 ('get', 3915),
 ('company', 3899),
 ('information', 3881),
 ('one', 3670)]
```

many of these tokens are one character long - remove these, perhaps with exception of numbers?

for the negative class:

```
[('_', 174714),
 ('language', 15162),
 ('university', 15147),
 ('subject', 14113),
 ('1', 13936),
 ('http', 12953),
 ('enron', 12349),
 ('com', 11705),
 ('e', 11580),
 ('2', 10140),
 ('one', 9584),
 ('0', 9432),
 ('00', 9220),
 ('information', 9021),
 ('mail', 8764),
 ('www', 8569),
 ('would', 8195),
 ('10', 8140),
 ('new', 8066),
 ('please', 8060)]
```

**Comment by Twigonometry** [comment]
Data from [f5a7ba](https://github.com/Twigonometry/Phishing-ML/commit/f5a7baded9c4499133ca73eb0c6c7a40b0f13e9d)

**Comment by Twigonometry** [comment]
Closed by [a38c54cd](https://github.com/Twigonometry/Phishing-ML/commit/a38c54cd8959f8b26fddca9d45cabb746200595c) - but we can certainly still experiment with being more/less ruthless. Hyperparam tuning can be done for
- min token length
- whether we remove numbers

Although it's commonly understood that more data gives better results, rather than hyperparameter searching
